pkg/registry/core/persistentvolumeclaim/strategy.go

```diff
@@ -97,6 +97,9 @@ func (persistentvolumeclaimStatusStrategy) PrepareForUpdate(ctx context.Context,
 	newPv := obj.(*api.PersistentVolumeClaim)
 	oldPv := old.(*api.PersistentVolumeClaim)
 	newPv.Spec = oldPv.Spec
+	if oldPv.Status.Conditions == nil {
```

Comment:
only if the feature gate is disabled

Reply:
and this needs a unit test

Reply:
Is there prior art for writing tests for `PrepareForUpdate` methods?

Reply:
I guess I could do a similar test like the other related PRs.

Reply:
sure, look for strategy_test.go files. In this case, the unit test would look a lot like the other PR's TestDrop___ tests, only the old object would never be nil

Reply:
Yep, I'm adding the test along with that change.